exercises/sum-of-multiples/sum_of_multiples_test.rb

```diff
@@ -2,48 +2,87 @@
 require 'minitest/autorun'
 require_relative 'sum_of_multiples'
 
-class SumTest < Minitest::Test
-  def test_sum_to_1
+# Common test data version: 1.0.0 72b1496
+class SumOfMultiplesTest < Minitest::Test
+  def test_multiples_of_3_or_5_up_to_1
+    # skip
     assert_equal 0, SumOfMultiples.new(3, 5).to(1)
```

Comment:
Probably unrelated to generator, but would it be interesting to allow `SumOfMultiples` to return something more Enumerator-like? 
`SumOfMultiples.new(3,5).take(1)`
cc: @kotp